blueprints/data-solutions/data-platform-minimal/03-curated.tf

```diff
@@ -75,6 +76,19 @@ module "cur-project" {
   }
 }
 
+module "cur-sa-0" {
```

Comment:
I'm not sure what is the purpose of this Service Account. But double checking with `landing` project and diagram, this would make sense if:
* there would be a GCS bucket in curated project (to match the diagram)
* this SA having `roles/storage.objectViewer` role

Reply:
This is a SA for external clients, for example a BI tool such as Looker.

Reply:
We're missing the curated gcs bucket in the outputs. ([Github doesn't allow comments on non-changed code](https://github.com/github/roadmap/issues/347))

Reply:
improved outputs